connect/cli/core/utils.py

```diff
@@ -3,10 +3,9 @@
 # This file is part of the Ingram Micro Cloud Blue Connect connect-cli.
 # Copyright (c) 2019-2022 Ingram Micro. All Rights Reserved.
 import os
-import re
 from collections import OrderedDict
-from distutils.version import StrictVersion
 
+from packaging.version import InvalidVersion, Version
```

Comment:
Can you also explain what is the reason for these changes? Do they correlate with hiding the warnings?

Reply:
No, this was the 3d warning group. In this case, in the warning they suggest to use this build-in module instead of the deprecated distutils.